examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>I would rephrase this sentence "We leveraged them for our RecSys2022 participations" as "We leveraged them for our RecSys practitioners."</p><p> </p><p>I find this sentence not clear <code>When the probabilities predicted by a Classification model are higher than its accuracy</code>. accuracy of the model do you mean? or number of samples classified correctly?</p><p> </p><p>I think this should be fixed as:</p><p> </p><p>α: Label smoothing hyper-parameter ( 0 &lt;= α &lt;= 1)</p><p> </p><p> </p><p>This sentence does not read well " we share the Embedding layer's weights which is used to convert the input to embeddings, as the softmax weights,  to convert hidden layer output to softmax layer output." . Please rephrase it, you can express it in 2 sentences to make it clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>